### PR TITLE
Use ISO 3166 lower case short names for countries

### PR DESCRIPTION
**Pull request by andylolz**
Refs #273.

### CODE REVIEW

File: xml/Country.xml [hunk 1]
```diff
@@ -972,553 +972,553 @@
...
             <name>
-                <narrative>WALLIS AND FUTUNA</narrative>
+                <narrative>Wallis and Futuna</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>EH</code>
             <name>
-                <narrative>WESTERN SAHARA</narrative>
+                <narrative>Western Sahara*</narrative>
```
Comment (andylolz) [inline xml/Country.xml]: Explanation for the asterisk here: https://www.iso.org/obp/ui/#iso:code:3166:EH

I’ll remove it.

**Comment by andylolz** [comment]
So, The Order Of Things is described here:
http://reference.iatistandard.org/203/codelists/codelist-management/#updates-to-non-embedded-codelists

> If a decision has been made to make a change to a non-embedded list the [IATI Standard discussion thread](http://discuss.iatistandard.org/c/iati-standard) will be notified (within 1 working day) about the change that is to be made and the date that the change will take effect (usually within 7 calendar days). The proposal will be marked as ‘Planned’ on the support forum. At this point people have the right to disagree. Discussion should take place via the [IATI Standard discussion thread](http://discuss.iatistandard.org/c/iati-standard). Unresolved matters can be taken to the following team meeting.
> 
> In most circumstances the decision to alter these codelists is straightforward. In rare cases, the team may decide to seek further clarification, or to consult further on whether or not to adopt a change. If so, a timetable outlining the process and time scales will be issued.
> 
> Once a change has been agreed, a GitHub issue will be created and will be linked to from the forum post.
> 
> When the change has been made and accepted into the master branch of the [Non-embedded Codelist repository](https://github.com/IATI/IATI-Codelists-NonEmbedded), the change will be ‘live’ to anyone who consumes these codelists directly from Github or via the API.
> 
> At this point the technical team will:
> 
> Update the [Non-Embedded Codelist changelog record](http://iatistandard.org/upgrades/nonembedded-codelist-changelog)
> Regenerate the IATI Standard Website.
> Mark the forum post as ‘Done’

So I think the first thing is to [create a discuss post](https://discuss.iatistandard.org/c/standard-management/non-embedded-codelist-amendments).

**Comment by andylolz** [comment]
I’ve added the discuss post here: https://discuss.iatistandard.org/t/added-update-organisationregistrationagency-codelist-from-org-id-guide/1301

**Comment by andylolz** [comment]
I’ve added the discuss post here: https://discuss.iatistandard.org/t/update-country-codelist-to-use-iso-3166-lower-case-short-names/1501

**Comment by samuele-mattiuzzo** [comment]
> I’ve added the discuss post here: https://discuss.iatistandard.org/t/update-country-codelist-to-use-iso-3166-lower-case-short-names/1501

thanks Andy!

**Pull request merged**